elements/lisk-p2p/src/p2p.ts

```diff
@@ -663,7 +617,10 @@ export class P2P extends EventEmitter {
 					return;
 				}
 
-				const wsPort: number = parseInt(queryObject.wsPort, BASE_10_RADIX);
+				const wsPort: number = parseInt(
+					queryObject.wsPort,
+					DEFAULT_BASE_10_RADIX,
```

Comment:
not sure about changing the name to DEFAULT_BASE_10_RADIX because there is only base 10 radix